ios/AppLovinMAXNativeAdViewManager.mm

```diff
@@ -61,4 +56,66 @@ - (UIView *)view
     }];
 }
 
+RCT_EXPORT_METHOD(updateAssetView:(nonnull NSNumber *)viewTag assetViewTag:(NSInteger)assetViewTag assetViewName:(NSString *)assetViewName)
+{
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
+        
+        UIView *view = viewRegistry[viewTag];
+        if ( ![view isKindOfClass: [AppLovinMAXNativeAdView class]] )
+        {
+            [[AppLovinMAX shared] log: @"Cannot find AppLovinMAXNativeAdView with tag %@", viewTag];
+            return;
+        }
+        
+        AppLovinMAXNativeAdView *nativeAdView = (AppLovinMAXNativeAdView *) view;
+        
+        if ( [assetViewName isEqualToString: @("TitleView")] )
```

Comment:
This is quite creative, where did you see in our codebase that we wrap strings with parentheses?